integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomain.java

```diff
@@ -1522,4 +1546,101 @@ private void waitForFileExistsInPod(String namespace, String podName, String fil
         .until(assertDoesNotThrow(() -> fileExistsInPod(namespace, podName, fileName),
             "fileExistsInPod failed with IOException, ApiException or InterruptedException"));
   }
+
+  /**
+   * Negative test case for creating a model-in-image domain without encryption secret created.
+   * The admin server service/pod will not be created.
+   * The error message should be logged in the operator log.
+   *
+   * @param domainUid the uid of the domain to be created
+   * @param domainNamespace namespace in which the domain will be created
+   */
+  private void createMiiDomainNegative(String domainUid, String domainNamespace) {
```

Comment:
This test seems unrelated to operator logging.

Reply:
This is to create a negative test case to generate SEVERE log msg in the operator log.

Reply:
Okay, I don't see that assertion. Is it already in checkServiceDoesNotExist()?

Reply:
No, checkServiceDoesNotExist() mainly to wait and check the service is deleted and not existing any more.

Reply:
Where is the check of the SEVERE in the log?

Reply:
It is in the test method testOperatorLogSevereMsg().